### PR TITLE
Fix failing chargebee test

### PR DESCRIPTION
**Pull request by AvoidMe**

### CODE REVIEW

**Comment by AvoidMe** [comment]
Test fixed: https://github.com/kuberlogic/kuberlogic/actions/runs/3093836824

File: modules/dynamic-operator/api/v1alpha1/webhook_suite_test.go [hunk 1]
```diff
@@ -190,3 +190,7 @@ var _ = AfterSuite(func() {
 	err := testEnv.Stop()
 	Expect(err).NotTo(HaveOccurred())
 })
+
+func useExistingCluster() bool {
```
Comment (telepenin) [inline modules/dynamic-operator/api/v1alpha1/webhook_suite_test.go:194]: If you added function why you don't refactoring places where the function is not used?

Comment (telepenin) [reply modules/dynamic-operator/api/v1alpha1/webhook_suite_test.go:194]: 80 line at least

Comment (AvoidMe) [reply modules/dynamic-operator/api/v1alpha1/webhook_suite_test.go:193]: fixed

**Review by telepenin — APPROVED** [review]

**Pull request merged**